src/test/java/com/yelp/nrtsearch/server/config/LuceneServerConfigurationTest.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2020 Yelp Inc.
```

Comment:
I noticed that spotless changes the year to 2020. Is that fine?

Reply:
Would be good to create an issue for this. I think updating the template will change all of the header. Maybe that is what we want, but I'm not sure.